src/app/dialogs/node-versions/node-versions.dialog.html

```diff
@@ -1,17 +1,39 @@
-<header mat-dialog-title>
+<header mat-dialog-title *ngIf="typeList">
   {{ 'VERSION.DIALOG_ADF.TITLE' | translate }}
 </header>
-<section mat-dialog-content>
-  <adf-version-manager
+<header mat-dialog-title *ngIf="!typeList">
+  {{ 'VERSION.DIALOG_ADF.TITLE' | translate }}
+</header>
+<section mat-dialog-content *ngIf="!typeList">
+  <adf-version-upload
+    id="adf-version-upload-button"
     [node]="node"
-    [showComments]="'adf-version-manager.allowComments' | adfAppConfig: true"
-    [allowDownload]="'adf-version-manager.allowDownload' | adfAppConfig: true"
-    (uploadError)="uploadError($event)"
+    [newFileVersion]="file"
+    (success)="handleUpload()"
+    (cancel)="handleUpload()"
+    (error)="onUploadError($event)"
   >
-  </adf-version-manager>
+  </adf-version-upload>
 </section>
-<footer mat-dialog-actions>
-  <button mat-button [mat-dialog-close]="true">
-    {{ 'VERSION.DIALOG_ADF.CLOSE' | translate }}
-  </button>
-</footer>
+<ng-container *ngIf="typeList">
+  <section mat-dialog-content>
+    <div class="adf-version-list-container">
+      <div class="adf-version-list-table">
+        <div>
```

Comment:
do you need this `<div>`? doesn't seem to do something

Reply:
modified with the requests